torch/_export/converter.py

```diff
@@ -338,6 +350,8 @@ def convert_graph_inputs(self):
                     self.fx_graph, name, self.is_top_level_graph()
                 )
             else:
+                if not is_valid_for_codegen(normalized_name):
+                    normalized_name = f"input_{normalized_name}"
```

Comment:
why dont we just call `normalize_name()` here? 
if you want things to look nicer, you could have `normalize_name` take in a prefix